api/utilities.py

```diff
@@ -52,21 +52,18 @@ def get_datetime_range(datetime_string: str | None) -> Tuple[Timestamp, Timestam
     return start_datetime, end_datetime
 
 
+@cached(ttl=600)
 async def get_current_parameter_names(ttl_hash=None):
```

Comment:
Get rid of the hash, as we now control TTL directly. The current setup will just fill up memory with no positive effect...